### DISCUSSION TITLE
End FeatHub for Feature Requests

### DISCUSSION THREAD

**Discussion by novaTopFlex** [Guake/guake#2253] [Q&A]
I am unsure why you, as part of the Guake community, are still associated with the FeatHub website after they had changed ownership to some Indonesian (possibly Malaysian?) entity. Is there a reason why your policies have not changed with the ownership of FeatHub?

**Comment by Davidy22** [comment]
Oh, looks like it's still in the feature request template, I took it out of the readme but missed that bit.

**Reply by Davidy22** [reply]
Struck in bfc94a7
